### PR TITLE
Add InstaDebit/iDebit

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Comment by dara123** [comment]
devs please review

File: spec/components/schemas/GatewayAccountConfig/InstaDebit.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+description: InstaDebit config
+allOf:
+-
+  $ref: "#/components/schemas/GatewayAccount"
+-
+  type: "object"
+  required:
+  - "credentials"
+  - "settings"
```
Comment (ashkarpetin) [inline spec/components/schemas/GatewayAccountConfig/InstaDebit.yaml]: You are missing `settings` properties or this is redundantю

Comment (slavcodev) [reply spec/components/schemas/GatewayAccountConfig/InstaDebit.yaml]: This comes from extended schema `GatewayAccount`

Comment (ashkarpetin) [reply spec/components/schemas/GatewayAccountConfig/InstaDebit.yaml]: But there is not `settings` in `GatewayAccount`..

**Pull request merged**